.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,24 @@
 ---
-name: 'Pull Request: New issue'
-about: Report and propose a fix for a (small) issue in Hazel
+name: 'Pull Request'
+about: Propose a fix implementation for a (small) issue in Hazel
 
 ---
 
-#### Describe the issue
+#### Describe the issue (if no issue has been made)
 A clear and concise description of what the issue is.
```

Comment:
I think we should keep some mention of the difference between the expected behavior and the current behavior.
```suggestion
A clear and concise description of what the issue is. Show us the difference between the expected behavior and the current behavior.
```

Reply:
More like this? It is not like they have to prove something.. (show us.. that seems a bit of an examination where you need gradings)
```suggestion
A clear and concise description of what the issue is. Explain the difference between the expected and the current behavior.
```

Reply:
Looks good to me.